### PR TITLE
Datalab: add metadata for server port

### PR DESCRIPTION
**Pull request by functicons**

### CODE REVIEW

File: datalab/datalab.sh [hunk 1]
```diff
@@ -149,8 +150,9 @@ EOF
 }
 
 function run_datalab() {
-  if docker run -d --restart always --net=host \
-    -v "${DATALAB_DIR}:/content/datalab" ${VOLUME_FLAGS} datalab-pyspark; then
+  if docker run -d --restart always -p "${DATALAB_PORT}:8080" \
```
Comment (medb) [inline datalab/datalab.sh]: Delete `:8080`

Comment (functicons) [reply datalab/datalab.sh]: It is not a mistake, it is a port mapping.

**Pull request merged**